src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const toNumbers = population.map((element) => {
+  return parseInt(element.textContent.replace(/,/g, ''));
```

Comment:
The arrow function inside the map method doesn't need to have a return statement since it's a single line of code. You can simplify it to `(element) => parseInt(element.textContent.replace(/,/g, ''))`.